shared/common-adapters/dice.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from 'react'
+import ReactDice from 'react-dice-complete'
+import 'react-dice-complete/dist/react-dice-complete.css'
+
+type Props = {
+  onRollDone: number => void,
+  values?: Array<number>,
+}
+
+export default class Dice extends React.Component<Props> {
+  reactDice: any
+  componentDidMount() {
+    this.reactDice.rollAll()
```

Comment:
I would just call this `_ref` or `_reactDiceRef`. Since that's more common for us and it'll be understood right away what this is.